src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,22 @@
+import "../../css/main.css";
+import "./footer.css";
+import { useAuth } from "../../Context";
```

Comment:
Suggestion: Try using absolute path instead of relative path. This will make your code look clean